services/121-service/src/utils/scope/createScopedRepositoryProvider.helper.ts

```diff
@@ -2,20 +2,22 @@ import { ScopedRepository } from '@121-service/src/scoped.repository';
 import { ScopedUserRequest } from '@121-service/src/shared/scoped-user-request';
 import { Provider, Scope } from '@nestjs/common';
 import { REQUEST } from '@nestjs/core';
-import { DataSource } from 'typeorm';
+import { DataSource, ObjectLiteral } from 'typeorm';
 
 // Todo make this strongly typed
```

Comment:
Should be // TODO: ? Or is // Todo also picked up by our todo extension?

Reply:
`todo extension` Do we have something like this? That todo is not part of these changes right?